components/tabs/tab.js

```diff
@@ -15,7 +15,6 @@ class Tab extends TabMixin(LitElement) {
 				white-space: nowrap;
 			}
 			:host(:first-child) .d2l-tab-text {
-				margin-inline-end: 0.6rem;
```

Comment:
This wasn't yet used but was inconsistent with the current LTR behaviour (which would have `margin-inline-end` as `0.5rem`)